packages/backend-core/src/middleware/auditLog.ts

```diff
@@ -1,6 +1,6 @@
-import { BBContext } from "@budibase/types"
+import { Ctx } from "@budibase/types"
 
-export default async (ctx: BBContext | any, next: any) => {
+export default async (ctx: Ctx, next: any) => {
```

Comment:
Can we not just delete this? It does nothing.

Reply:
It is a place holder that is overriden by pro - its one of the oddities that we have to deal with due to the closed source submodule.